### ISSUE TITLE
NTR: Spinal cord biopsy: granulomas (Lab: pathology: biopsies & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9447]
New term request Spinal cord biopsy: granulomas (elthat_211222141925):

**Comment by pnrobinson** [comment]
@MickeySegal
Should we specify what part of the spinal cord? I see several references to the granulomatosis localized in the leptomeninges of the spinal cord, is this what is meant?

**Comment by MickeySegal** [comment]
We only have this for https://omim.org/entry/619381, which refers to "central nervous system inflammation with granulomatous lesions of the spinal cord".

**Comment by pnrobinson** [comment]
In this case it is epidural on the cervical spinal cord http://www.ajnr.org/content/24/1/18
here also
https://www.frontiersin.org/articles/10.3389/fimmu.2018.01258/full
however, in this paper it is intramedullary
Cervical and thoracic spine MRI of Patient 4 taken at 26 years of age showing a signal enhancing intramedullary space-occupying lesion through nearly the entire length of the cervical and thoracic spinal cord. One representative image out of 6 acquired images is shown. All images revealed signs of inflammation. 
(Extended figure 2b).

**Comment by MickeySegal** [comment]
Also https://www.omim.org/entry/619381 "central nervous system inflammation with granulomatous lesions of the spinal cord that responded to high-dose steroids".

**Comment by pnrobinson** [comment]
@MickeySegal 

Adding new term: 
- ID: HP:6000257
- label: Spinal cord granuloma
- parents: [Abnormal spinal cord morphology (HP:0002143)](https://hpo.jax.org/app/browse/term/HP:0002143)
- pmid: PMID:12533321
- def: An aggregation of macrophages (forming in response to chronic inflammation) localized in or on the spinal cord.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9447
- synonyms:
